src/main/kotlin/com/hibob/academy/service/SessionService.kt

```diff
@@ -0,0 +1,35 @@
+package com.hibob.academy.service
+
+import com.hibob.academy.resource.LoginUser
+import io.jsonwebtoken.Jwts
+import io.jsonwebtoken.SignatureAlgorithm
+import org.springframework.stereotype.Component
+import java.time.LocalDateTime
+import java.time.ZoneOffset
+import java.util.Date
+
+@Component
+class SessionService {
+
+    companion object {
+
+        const val SECRET_KEY: String = "secrethashkeyshahar12323445665713124jadnasdkllFSDFNSD89SLJNLASJNLD"
+    }
+
+    fun createJwtToken(user : LoginUser): String {
+
+        val now : Date = Date.from(LocalDateTime.now().toInstant(ZoneOffset.UTC))
+        val expirationDate : Date = Date.from(LocalDateTime.now().plusDays(1).toInstant(ZoneOffset.UTC))
```

Comment:
it's no biggie, but if you already created the now object, you might as well reuse it on this line